sdk/src/main/java/com/mapbox/maps/ResourceOptionsManager.kt

```diff
@@ -20,23 +20,41 @@ class ResourceOptionsManager(
     resourceOptions = resourceOptions.toBuilder().apply(block).build()
   }
 
+  /**
```

Comment:
ResourceOptionsManager should be a data class to be aligned with CredentialsManager.